src/IO/WriteHelpers.h

```diff
@@ -1183,13 +1183,13 @@ inline void writeBinaryEndian(T x, WriteBuffer & buf)
 }
 
 template <typename T>
-inline void writeBinaryLittleEndian(T x, WriteBuffer & buf)
+inline void writeBinaryLittleEndian(const T & x, WriteBuffer & buf)
```

Comment:
I don't understand why this line changed, but line 1179 isn't.
Most likely, no changed are needed in this file.

Reply:
That variant takes its first parameter by value so that `transformEndianness` can work its magic when necessary (i.e. when native endianness and desired endianness don't match).

Reply:
I'll just get rid of the `const &` and move on.